osprey_ui/src/components/event_stream/EventStream.module.css

```
@@ -18,6 +18,34 @@
   outline: none;
 }
 
+/* react-virtualized renders a scrolling div with class `ReactVirtualized__List`.
+   On macOS the default scrollbar auto-hides until you're actively scrolling,
+   which makes it hard to skim long result sets — flagged in PR #367. Force a
+   thin always-visible bar in the theme colors. */
+.virtualizedList :global(.ReactVirtualized__List) {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)